src/MatBlazor/Components/MatCard/MatCardContent.razor

```diff
@@ -18,7 +18,6 @@
          protected override async Task OnInitializedAsync()
          {
              await base.OnInitializedAsync();
-             ClassMapper.Add("mdc-card__primary-action");
```

Comment:
@Eonasdan Did you try to pull a little sneaky here? ;- ) (Saw your Issue but let's take discussion on this there instead and make this a separate PR)

Reply:
Not sneaky lol. It was just causing issues in my application. I remove this line from the pr and do branches in my own pr to keep stuff separate.

Reply:
done